lua/lvim/core/which-key.lua

```diff
@@ -68,7 +70,7 @@ M.config = function()
       ["q"] = { "<cmd>q!<CR>", "Quit" },
       ["/"] = { "<cmd>lua require('Comment').toggle()<CR>", "Comment" },
       ["c"] = { "<cmd>BufferClose!<CR>", "Close Buffer" },
-      ["f"] = { "<cmd>Telescope find_files<CR>", "Find File" },
+      ["f"] = { custom_finders.find_project_files, "Find File" },
```

Comment:
I had no idea that this actually works. I think it's better to not require that file too early, since it might cause errors on new installations.

```suggestion
      ["f"] = { require ("lvim.core.telescope.custom-finders").find_project_files, "Find File" },
```

Reply:
✅